dotnet-authserver/src/TeacherIdentity.DevBootstrap/Csv/UserReaderMap.cs

```diff
@@ -0,0 +1,20 @@
+using CsvHelper.Configuration;
+using TeacherIdentity.AuthServer.Models;
+
+namespace WorkforceDataApi.DevUtils.Csv;
```

Comment:
Could we change this to match other files in this project?

Reply:
Doh - done!